<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QWidget" name="loginpage">
    <widget class="QLineEdit" name="login_LE">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入你的昵称</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_PB">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>280</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="selectpage">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="selectLayout">
      <item>
       <widget class="QPushButton" name="s_server_PB">
        <property name="text">
         <string>接受对方连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="s_client_PB">
        <property name="text">
         <string>连接对方</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="serverpage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>正在等待连接</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="serverRe_PB">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="clientpage">
    <widget class="QPushButton" name="client_PB">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>请求连接</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="client_LE">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入对方的ip地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clientRe_PB">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>350</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gamepage">
    <widget class="QLabel" name="p1name_LE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="shootp1_PB">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>射向自己</string>
     </property>
    </widget>
    <widget class="QPushButton" name="shootp2_PB">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>射向对面</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="messagebox">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>300</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="p1tools_F">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="p2tools_F">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="p2name_LE">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="bloodbox">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="overpage">
    <widget class="QLabel" name="overtext">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="again_PB">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>再来一局</string>
     </property>
    </widget>
    <widget class="QPushButton" name="return_PB">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>340</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
